packages/react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -97,284 +93,231 @@ export const FocusTrapZone: React.FunctionComponent<IFocusTrapZoneProps> & {
     'data-is-focus-trap-zone-bumper': true,
   } as React.HTMLAttributes<HTMLDivElement>;
 
-  const focus = React.useCallback(() => {
+  const focusElementAsync = React.useCallback((element: HTMLElement) => {
```

Comment:
This was a helper function that existed in v7, and its logic was moved elsewhere in the FC conversion, but it seemed better to me to bring it back as a separate function to simplify various places and reduce possible bugs from missing validation